test/resources/machine-set.yaml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
I can't see where this is used

Reply:
I currently use this manually when standing up a kubemark cluster so can we keep it please.

Reply:
in the CI to create machine set

Reply:
Can we move the CI code that instantiate this into here so upcoming changes can be done atomically in one repo?

Reply:
We can't. The test assumes a machine set already exists. The goal is to eventually run it over openshift as well. Thus, make it uniform.

Reply:
so before the actual validation runs, why can't we instantiate the machineSet here?

Reply:
My goal was to keep cao unaware of underlying machines or machinesets. Cao is responsible only for rendering manifests specified by ClusterAutoscaler and MachineAutoscaler resources. The less knowledge we expose, the better. Thus, the reason to have machineset created independently of running the tests. We need to distinguish between expectations of what always has to be present in the cluster and what the tests are responsible for creating (and deleting).